packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -30,6 +30,14 @@ describe('request-queue tests', () => {
     callbacks,
   });
 
+  const getDelayedPromise = () => {
```

Comment:
You have a very similar test method with a different name in another test file:
https://github.com/GoogleChrome/workbox/blob/master/packages/workbox-background-sync/test/browser/background-sync-queue.js#L18-L24

I wonder if it makes sense to wait until my changes in #690 get merged so you don't have to repeat yourself here, and you can just import a common utility method that does this.

@gauntface WDYT?